packages/react-app/components/EventRow.jsx

```diff
@@ -16,20 +17,26 @@ const EventRow = ({ event }) => {
     userAddress = buidlGuidlAddress;
   }
 
+  const { baseBlueColor } = useCustomColorModes();
+  const [eventTitle, eventDescription] = eventDisplay(event);
+
   return (
-    <Tr>
-      <Td>
-        <NextLink href={`/builders/${userAddress}`} passHref>
-          <Link pos="relative">
-            <Address address={userAddress} w="12.5" fontSize="16" />
-          </Link>
-        </NextLink>
-      </Td>
-      <Td whiteSpace="nowrap">
-        <DateWithTooltip timestamp={event.timestamp} />
-      </Td>
-      <Td>{eventDisplay(event)}</Td>
-    </Tr>
+    <Box backgroundColor={baseBlueColor} mb={4} p={8}>
+      <HStack spacing={8} align="flex-start" justify="space-between">
+        <HStack spacing={2.5} align="flex-start">
+          <NextLink href={`/builders/${userAddress}`} passHref>
+            <Link pos="relative" mt="-5.5px">
```

Comment:
Not proud of this... haha But not sure about a better way of aligning this

Reply:
Lol, I am not sure either how would we handle this without changing Address component 

But I think its good and works nicely on mobile too !